pkgs/applications/misc/barrier/default.nix

```diff
@@ -14,11 +15,16 @@ mkDerivation rec {
   };
 
   buildInputs = [ cmake curl xorg.libX11 xorg.libXext xorg.libXtst avahiWithLibdnssdCompat qtbase ];
```

Comment:
Could you also move `cmake` to nativeBuildInputs please? It's only needed at build time.

Reply:
Done.